tests/without_mdx_unimoji_extension.py

```diff
@@ -0,0 +1,16 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from unittest import main, TestCase
+
+
+class Test(TestCase):
+    def test_markdown_without_emoji_extension(self):
```

Comment:
Could short tests like these be merged into a single file? It seems somewhat redundant to have them in separate files when the method name itself seems to convey action sufficiently.